db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f"{self.name}"
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        "what kind of bonus players can get", max_length=255,
+    )
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return f"{self.name}"
```

Comment:
Why do you need f-string here?

Reply:
Because if I write self.name only, it shows warning - Expected type 'str', got 'CharField' instead

Reply:
It is PyCharm warning, not python warning